src/ng2-smart-table/ng2-smart-table.component.ts

```diff
@@ -1,19 +1,16 @@
-import {
-  Component, Input, Output, SimpleChange, EventEmitter,
-  OnChanges
-} from '@angular/core';
+import { Component, Input, Output, SimpleChange, EventEmitter, OnChanges, ViewEncapsulation } from '@angular/core';
 
 import { Grid } from './lib/grid';
 import { DataSource } from './lib/data-source/data-source';
 import { Row } from './lib/data-set/row';
-
 import { deepExtend } from './lib/helpers';
 import { LocalDataSource } from './lib/data-source/local/local.data-source';
 
 @Component({
   selector: 'ng2-smart-table',
   styleUrls: ['ng2-smart-table.scss'],
   templateUrl: 'ng2-smart-table.html',
+  encapsulation: ViewEncapsulation.None
```

Comment:
you can use `/deep/` instead of `encapsulation: ...`. scss code for example:
`
/deep/ ng2-smart-table {
    table.ng2-smart-table {
       ...
    }
}
`